src/resourcemgr/fileBrowser.js

```diff
@@ -274,7 +274,25 @@ export default function (options) {
             dataType: 'json',
             data: _.merge(parameters, options.params, { childrenOffset: (selectedClass.page - 1) * selectedClass.childrenLimit }),
             noToken: true,
-        }).then(response => response.data);
+        })
+        .then(response => response.data)
+        .then(response => {
+            let permissions = {
+                read: false,
+                write: false
+            }
+            if (response.permissions) {
```

Comment:
If we do not have the `response.permissions`, read and write should be true by default.

Reply:
Done